input/ig.xml

```diff
@@ -396,135 +396,135 @@
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/pathresult-covid-1"/>
 			</reference>
 			<name value="Observation - COVID Pathology Result (Not detected)"/>
-			<description value="COVID Pathology Result (Not detected)&#xa;&#xa;Patient: Mr. Li Wang"/>
+			<description value="Shows an example of a COVID-19 PCR test result indicating 'Not detected' for the *AU Core Diagnostic Result - Pathology* profile, with notes on test accreditation and evaluation status. Patient: Li Wang."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-diagnosticresult-path"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/resprate-1"/>
 			</reference>
 			<name value="Observation - Respiration rate"/>
-			<description value="Observation of respiration rate&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an example of a respiration rate observation for the *AU Core Respiration Rate* profile. Patient: Mia Leanne Banks."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-resprate"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/smokingstatus-current-smoker"/>
 			</reference>
 			<name value="Observation - Current smoker"/>
-			<description value="Observation of smoking status of current smoker&#xa;&#xa;Patient: Mr. Li Wang&#xa;&#xa;Performer: Dr. Bob Bobrester, GP"/>
+			<description value="Shows an example of an observation of smoking status of current smoker for the *AU Core Smoking Status* profile. Patient: Mr. Li Wang."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-smokingstatus"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/waistcircum-1"/>
 			</reference>
 			<name value="Observation - Waist Circumference"/>
-			<description value="Waist Circumference&#xa;&#xa;Patient: Mia LEANNE BANKS"/>
+			<description value="Shows an example of a waist circumference measurement result for the *AU Core Waist Circumference* profile. Patient: Mia Leanne Banks."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-waistcircum"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Observation/observation-masked"/>
 			</reference>
-			<name value="Observation - Data suppressed for category, code and effectiveDate"/>
-			<description value="Observation with masked category, code and effectiveDate&#xa;&#xa;Patient: Mr. Ronny Irvine"/>
+			<name value="Observation - Data suppressed for category, code, effective[x] and value[x]"/>
+			<description value="Shows an example of suppressed data (category, code, effectivePeriod and valueQuantity) for the *AU Core Diagnostic Result Observation* profile. Patient: Ronny Lawrence Irvine."/>
```

Comment:
The suppressed data change is not applied here yet.

Should say: Shows an example of an observation with suppressed data (category, code, effectivePeriod, and valueQuantity) where the data requester has access rights to know that data is suppressed for the *AU Core Diagnostic Result Observation* profile. Patient: Ronny Lawrence Irvine.

Reply:
Done, sorry - missed this one earlier